creator-node/src/routes/users.js

```diff
@@ -16,23 +16,29 @@ const CHALLENGE_PREFIX = 'userLoginChallenge:'
 
 module.exports = function (app) {
   app.post('/users', handleResponse(async (req, res, next) => {
-    let walletAddress = req.body.walletAddress
+    const walletAddress = req.body.walletAddress
+    const isCreatorNode = req.body.isCreatorNode === true
+
     if (!ethereumUtils.isValidAddress(walletAddress)) {
-      return errorResponseBadRequest('Ethereum address is invalid')
+      return errorResponseBadRequest('Invalid request body params')
     }
 
-    walletAddress = walletAddress.toLowerCase()
+    const walletPublicKey = walletAddress.toLowerCase()
 
+    // do nothing if CNodeUser already exists
     const existingUser = await models.CNodeUser.findOne({
-      where: {
-        walletPublicKey: walletAddress
-      }
+      where: { walletPublicKey }
     })
-    if (existingUser) {
-      return successResponse() // do nothing if user already exists
+
+    // if CNodeUser doesn't already exist, create it
+    if (!existingUser) {
+      await models.CNodeUser.create({
+        walletPublicKey,
+        isCreatorNode
```

Comment:
i feel like we should have an spId field in this table if creator node. that would make lookups between cnodes much easier.

Reply:
per discussion, going to standardize all to spID